crates/orchestrator/src/setup/mod.rs

```diff
@@ -40,21 +40,21 @@ pub async fn setup_cloud(setup_cmd: &SetupCmd) -> color_eyre::Result<()> {
             s3.setup(&StorageValidatedArgs::AWSS3(aws_s3_params.clone())).await?
         }
     }
-    println!("Data storage setup completed ✅");
+    tracing::info!("Data storage setup completed ✅");
 
     // Queues
-    println!("Setting up queues. ⏳");
+    tracing::info!("Setting up queues. ⏳");
     let queue_params = setup_cmd.validate_queue_params().expect("Failed to validate queue params");
     match queue_params {
         QueueValidatedArgs::AWSSQS(aws_sqs_params) => {
             let sqs = Box::new(SqsQueue::new_with_args(aws_sqs_params, aws_config));
             sqs.setup().await?
         }
     }
-    println!("Queues setup completed ✅");
+    tracing::info!("Queues setup completed ✅");
 
     // Cron
-    println!("Setting up cron. ⏳");
+    tracing::info!("Setting up cron. ⏳");
```

Comment:
is this working? i think it doesn't work in setup command becuae tracing isn't initialiosed

Reply:
there was a tracing here in this [file](https://github.com/madara-alliance/madara-orchestrator/pull/179/files/982aed98dfafe96f5012acceea45198e051d2b77..a7e5947fd726e14972baace398f9ce2dd7a05f0f#diff-00097fef3d0ada74431d3dd28454a1e8eb65b20de0accc4ba89ef20d73d3f42dR89), I thought it was initialized.